.changeset/wicked-months-listen.md

```diff
@@ -0,0 +1,5 @@
+---
+'@roadiehq/backstage-plugin-datadog': patch
```

Comment:
This looks to be introducing new functionality so we should be using minor version here.